public/app/core/components/sidemenu/DropDownChild.tsx

```diff
@@ -14,12 +14,24 @@ const DropDownChild: FC<Props> = props => {
     margin-right: ${theme.spacing.sm};
   `;
 
+  const childrenMenu = child.children
+    ? child.children.map((child, index) => {
+        return (
+          <>
+            <DropDownChild child={child} key={`${child.url}-${index}`} />
+          </>
+        );
+      })
+    : null;
+
+  const renderChildren = !!child.children;
   return (
     <li className={listItemClassName}>
       <a href={child.url}>
         {child.icon && <Icon name={child.icon as IconName} className={iconClassName} />}
         {child.text}
       </a>
+      {renderChildren ? <ul className="sidemenu dropdown-menu--sidemenu submenu">{childrenMenu}</ul> : null}
```

Comment:
Still not sure about the convention here, but is `renderChildren && <ul>...</ul>` an option?

Reply:
@fabio-silva Yes, if renderChildren is a boolean.

Reply:
@atymchuk  right 👍